doc/examples/setup_assistant/setup_assistant_tutorial.rst

```diff
@@ -1,383 +1,524 @@
+  files to be generated tab and you can click on each of them for a
+  description of what they do. For more information on the generated files,
+  see :doc:`the Configuration section in the documentation </doc/examples/examples>`.
 
-.. image:: setup_assistant_panda_done.png
+.. image:: setup_assistant_done.png
    :width: 700px
 
-* Congratulations!! - You are now done generating the configuration
-  files you need for MoveIt
+Congratulations! You are now done generating the configuration files you need for MoveIt.
 
+Build the panda_moveit_config package and run the demo
+------------------------------------------------------
+To build only the generated ``panda_moveit_config`` package and run the demo, follow these steps. ::
 
-What's Next
----------------
+   cd ~/ws_moveit2
+   colcon build --packages-select panda_moveit_config
+   source install/setup.bash
 
+Start the MoveIt demo to interactively plan and execute motions for the robot in RViz. ::
 
-The MoveIt RViz plugin
+   ros2 launch panda_moveit_config demo.launch.py
 
-* Start looking at how you can use the generated configuration files
-  to play with MoveIt using the
-  :doc:`MoveIt RViz Plugin </doc/tutorials/quickstart_in_rviz/quickstart_in_rviz_tutorial>`.
+Check out this `brief YouTube video <https://youtu.be/f__udZlnTdM>`_ for an example of how to
+command the robot to move to the pre-defined ``ready`` pose and execute ``open`` and ``close`` motions on the hand.
 
-Setup IKFast Inverse Kinematics Solver
+What's Next
+-----------
 
-* A faster IK solver than the default KDL solver, but takes some
-  additional steps to setup:
-  :doc:`Kinematics/IKFast </doc/examples/ikfast/ikfast_tutorial>`
+Get Started with MoveIt Motion Planning using RViz
 
-Additional Reading
----------------------
+* Learn how to use the generated configuration files to plan and visualize motion with MoveIt in RViz.
+  Check out the :doc:`MoveIt Quickstart in Rviz tutorial </doc/tutorials/quickstart_in_rviz/quickstart_in_rviz_tutorial>` for a step-by-step guide.
+
+Write Your First C++ MoveIt Application
+
+* Write your first C++ application using MoveIt with :doc:`this tutorial </doc/tutorials/your_first_project/your_first_project>`,
+  and familiarize yourself with the ``MoveGroupInterface`` and use it to plan, execute, and visualize motion plans for your robot from :doc:`this example </doc/examples/move_group_interface/move_group_interface_tutorial>`.
+
+URDF vs SRDF: Understand the Differences
 
 * See the :doc:`URDF and SRDF </doc/examples/urdf_srdf/urdf_srdf_tutorial>` page for more
   details on the components of the URDF and SRDF mentioned in this tutorial.
+
+Setup Custom Inverse Kinematics Solvers
+
+* Faster IK solvers than the default KDL solver are available.
+  For more information, refer to the :doc:`IKFast Kinematics SolverIKFast </doc/examples/ikfast/ikfast_tutorial>` and :doc:`TRAC-IK Kinematics Solver </doc/examples/trac_ik/trac_ik_tutorial>`.
```

Comment:
As per my previous comment on IK plugins, try use more neutral wording. Specifically,

* KDL is actually pretty fast already, so switching away from KDL is usually to get better performance especially in the face of constraints. IKFast is the actual faster exception because it can generate analytical solutions, but other solvers that use optimization such as TRAC-IK, Bio-IK, and pick_ik are actually slower.
* Also, TRAC-IK is not yet available in ROS 2 so I would advise against remove it and point to `pick_ik` instead if you'd like.

Reply:
trac_ik and bio_ik are not slower than KDL! See https://bitbucket.org/traclabs/trac_ik/src/master/

Reply:
oh wow, that's surprising considering TRAC-IK is literally KDL + an optimization based solver in parallel. I guess the authors made some improvements to the Orocos KDL package...

Reply:
TRAC-IK is an enhancement of KDL + an optimization based solver. This enhancement is referred to as KDL-RR (Random Restarts) by the authors. Unlike the default KDL solver, KDL-RR uses a time-based run loop instead of a fixed number of iterations. Additionally, it utilizes random restarts (seeds) to overcome local minima when detected.

@sea-bass, I updated the tutorial to use neutral wording when alternative IK solvers are mentioned, and took a note this point may need reconsideration after more IK plugins are available.